cmd/pr-reminder/main.go

```diff
@@ -424,3 +443,12 @@ func messageUser(user user, slackClient slackClient) error {
 
 	return kerrors.NewAggregate(errors)
 }
+
+// appendLabels checks the PR's labels and returns a message if PR is held,
+// or empty string if the PR is not held.
+func appendLabels(labels []string) string {
+	if len(labels) == 0 {
+		return " " //if PR has no labels, this adds nothing to the slack message
+	}
+	return fmt.Sprintf(":warning: This PR is labeled: *%v*", strings.Join(labels[:], ", "))
```

Comment:
I like the idea of the `warning` emoji, but ultimately we might want to include other labels here and it may be a bit too heavy. It appears there is a `label` emoji, so lets try that.
```suggestion
	return fmt.Sprintf(":label: labeled: *%v*", strings.Join(labels[:], ", "))
```